Created an Addons section

DevilsDesigns opened the pull request:
I originally added a github repo stars in plugins. I decided to add an addons repo with all the possible addons not already listed in jellyfins docs. I figured this was best since it didnt fit any of the other categories and to make it simpler for people to find what they want in the least amount of clicks.

TimGels commented on general/server/plugins/addons.md:
```diff
@@ -0,0 +1,85 @@
+### JF-Secure
+
+#### Description
+A turn key Caddy Reverse Proxy Webserver setup for Jellyfin without any codiung required.
```
```suggestion
A  turn-key reverse proxy tool that makes use of Caddy. It will setup a reverse proxy for Jellyfin with the goal of being easy to use, even for novice users.
```

I do not know the project deeply as this is the first time that I heard it. But what do you think of this description?

DevilsDesigns replied:
okay i made all the grammar corrections thank you for the advice it does sound and look a lot better.

TimGels reviewed (commented):
I currently don't have time to read all of it. But I found a couple of spelling errors and added some suggestions which you might want to look into. 

Furthermore, I think adding a general introduction to this page to let the user know what it is about and what addonss are might also be a nice addition to consider.

whiteowl3 commented on general/server/plugins/addons.md:
```diff
@@ -0,0 +1,86 @@
+title: Addons
+---
+### Introduction
+This page is all about any addition ways to integrate your Jellyfin install with other addition addons to make your experience more enjoyable and entertaining.
```
```suggestion
This page is all about ways to integrate your Jellyfin installation with extra add-ons to make your experience more enjoyable and entertaining.
```

whiteowl3 commented:
#688 
Please consider using "add-ons" as this is technically correct in English.

DevilsDesigns commented:
made the required edits

> #688 Please consider using "add-ons" as this is technically correct in English.

TimGels commented on general/server/plugins/add-ons.md:
```diff
@@ -0,0 +1,86 @@
+title: Add-ons
+---
+### Introduction
+This page is all about any addition ways to integrate your Jellyfin install with other addition add-ons to make your experience more enjoyable and entertaining.
```
```suggestion
This page is all about additional ways to integrate your Jellyfin install with other available add-ons to make your experience more enjoyable and entertaining.
```

TimGels commented on general/server/plugins/add-ons.md:
```diff
@@ -0,0 +1,86 @@
+### TV Intro Detection
+
+#### Description
+This is an automated way to skip all your TV Show introductions like netflix does. Unfortuntly it does not do outros yet, but its a great start.
```
```suggestion
This is an automated way to skip the introduction of your tv series, just like you are used to from the likes of Netflix.
```
I think omitting the unfortunately results in a higher positive feeling when reading the text.

TimGels commented on general/server/plugins/add-ons.md:
```diff
@@ -0,0 +1,86 @@
+
+**Links:**
+
+* [GitHub](https://github.com/jellyfin/jellyfin-plugin-emailnotifications)
```
```suggestion
* [GitHub](https://github.com/mueslimak3r/tv-intro-detection)
```

Shouldn't this be pointing to the tv intro detection repository?

TimGels commented on general/server/plugins/add-ons.md:
```diff
@@ -0,0 +1,86 @@
+### Emby2Jelly
+
+#### Description
+This is a good way to move jellyfin from one server to another host of jellyfin, or move from Emby to Jellyfin without losing data metadata critical data.
```
```suggestion
This add-on provides a way to migrate users and watched content from Emby to Jellyfin.
```
They only state that it works when migrating from Emby to Jellyfin. Did you test this with moving Jellyfin as well? 

Also, they state that the Jellyfin version tested is 10.5.4 which is quite old already. Is this tested on 10.8.x?

TimGels commented on general/server/plugins/add-ons.md:
```diff
@@ -0,0 +1,86 @@
+uid: server-plugins-Add-ons
+title: Add-ons
+---
+### Introduction
```
```suggestion
# Add-ons
```
Other pages start with the title and have the description immediately under this header.

TimGels commented on general/server/plugins/add-ons.md:
```diff
@@ -0,0 +1,86 @@
+* [Github](https://github.com/koying/jellyfin_ha)
+* [Latest Release](https://github.com/koying/jellyfin_ha/releases)
+
+### Jellyfin Party Extensions
```
```suggestion
### Jellyfin Party Extension
```

TimGels commented:
None of the projects listed here get installed as a plugin, so they do not fit in with the others on the plugin section, right?

I also wonder if "add-on" is a term that captures these projects accurately with how they are related to and used with Jellyfin, but that could be just me wondering for no reason, as I currently do not know a better alternative name myself. Might be interesting what other contributors think of this section and the name.

DevilsDesigns commented:
> None of the projects listed here get installed as a plugin, so they do not fit in with the others on the plugin section, right?
> 
> I also wonder if "add-on" is a term that captures these projects accurately with how they are related to and used with Jellyfin, but that could be just me wondering for no reason, as I currently do not know a better alternative name myself. Might be interesting what other contributors think of this section and the name.

Correct thats why i made a new section because they wouldnt fix in the plugins section. I was figuring if it wasnt add-ons it would have been extensions but that i feel doesnt fit as good as add-ons because that means their something you can addon to your jellyfin install. Also I made the required edits.

TimGels commented:
> > None of the projects listed here get installed as a plugin, so they do not fit in with the others on the plugin section, right?
> > I also wonder if "add-on" is a term that captures these projects accurately with how they are related to and used with Jellyfin, but that could be just me wondering for no reason, as I currently do not know a better alternative name myself. Might be interesting what other contributors think of this section and the name.
> 
> Correct thats why i made a new section because they wouldnt fix in the plugins section. I was figuring if it wasnt add-ons it would have been extensions but that i feel doesnt fit as good as add-ons because that means their something you can addon to your jellyfin install. Also I made the required edits.

I asked around with other contributors and names like "integrations", "third-party software", "third-party integrations" and "works with Jellyfin" also came up as possible alternative names. I personally like third-party software or third-party integrations myself (but that is just my personal opinion). These names capture this section better I think. What do you think of these names?

DevilsDesigns commented:
> > > None of the projects listed here get installed as a plugin, so they do not fit in with the others on the plugin section, right?
> > > I also wonder if "add-on" is a term that captures these projects accurately with how they are related to and used with Jellyfin, but that could be just me wondering for no reason, as I currently do not know a better alternative name myself. Might be interesting what other contributors think of this section and the name.
> > 
> > 
> > Correct thats why i made a new section because they wouldnt fix in the plugins section. I was figuring if it wasnt add-ons it would have been extensions but that i feel doesnt fit as good as add-ons because that means their something you can addon to your jellyfin install. Also I made the required edits.
> 
> I asked around with other contributors and names like "integrations", "third-party software", "third-party integrations" and "works with Jellyfin" also came up as possible alternative names. I personally like third-party software or third-party integrations myself (but that is just my personal opinion). These names capture this section better I think. What do you think of these names?

I love your suggestions way more professional and way easier to understand the naming. I changed the Documentations titles to Third-Party Integrations and moved it to the contributions section of the documentation. I also deleted the original Add-ons Section in Plugins documentation in place for the Contributions/Third-Party-Integrations.md. Let me know if we can push this through or any other edits need to be made. I also changed all forms of Add-ons text to Third-Party Integrations

TimGels commented:
> I love your suggestions way more professional and way easier to understand the naming. I changed the Documentations titles to Third-Party Integrations and moved it to the contributions section of the documentation. I also deleted the original Add-ons Section in Plugins documentation in place for the Contributions/Third-Party-Integrations.md. Let me know if we can push this through or any other edits need to be made. I also changed all forms of Add-ons text to Third-Party Integrations

I don't see a `contributing` uid being used anywhere else. This one also isn't added to the toc file. So it will not be visible. I also don't know if I like this place for third-party software as it is more geared towards getting people to understand how they can contribute to Jellyfin, rather than seeing what software works with Jellyfin. I suggest you to ask what other people's opinions are on a fitting place for this page.

DevilsDesigns commented:
i made all the gramatical corrections. I also added it to toyc and its referenced. Could be more specific about asking the community. Who should I ask, and where should? Let me know if anything other ideas you may have?

TimGels commented:
> i made all the gramatical corrections. I also added it to toyc and its referenced. Could be more specific about asking the community. Who should I ask, and where should? Let me know if anything other ideas you may have?

You are active on Matrix / Discord in the development channel. I think thats a good place to ask. I also asked yesterday and i got a response saying that it could also possibly fit in its own category all together. Which I agree with too.

whiteowl3 commented on general/contributing/Third-Party Integrations.md:
```diff
@@ -0,0 +1,87 @@
+### Jellyfin Party Extension
+
+#### Description
+Have you ever wanted to have all your friends watch a movie or tv show together without having to give them accss to the Jelyfin UI. Well here it is. Just pull up a chrome make a link start the movie and stream with all your friends through your chrome browser to theirs.
```
"Jelyfin UI" >> "Jellyfin UI"

TimGels replied:
great catch!
